CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 # 0.10.0 - Next Release In Development
 
 * **BREAKING:** Using `.equalTo(<EdgeInsets>)` will no longer multiply the constants on the right/bottom edge by -1. This does not affect `.offset(<EdgeInsets>)`
```

Comment:
It's actually quite weird we even allow the equalTo to be UIEdgeInsets because I was thinking that doing edges.equalTo(UIEdgeInsetsMake(10,10,20,20)) would lock top = 10, left = 10, bottom = 20, right = 20 but it actually insets it from the superview by that amount.